src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -64,16 +79,16 @@ public Startup(IConfiguration config, ILogger<Startup> logger, IWebHostEnvironme
             }
 
             _logger.LogInformation($"UseSandbox is: {_useSandbox.ToString()}");
+            
+            Configuration = _config.Get<ApiConfiguration>();
```

Comment:
I notice the difference here from the need to cast to IConfiguration above - which is correct?